launcher/image/test/scripts/test_debug_alive.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+source util/read_serial.sh
```

Comment:
No need to import `read_serial.sh` as it is not used in this script.

Reply:
Removed